assets/javascripts/card-form-customization.js

```diff
@@ -0,0 +1,93 @@
+function setInitialValues() {
```

Comment:
Should we rename to `setDesignFormValues`?

Reply:
The function set initial value to input (i.e color input, select input).

Reply:
To pair with `getDesignFormValues`

Reply:
Ahh got it. Thanks will update.